examples/jsm/renderers/common/StorageBufferAttribute.js

```diff
@@ -0,0 +1,21 @@
+import { InstancedBufferAttribute } from 'three';
+
+class StorageBufferAttribute extends InstancedBufferAttribute {
+
+	constructor( array, itemSize ) {
+
+		super( array, itemSize );
+
+		this.isStorageBufferAttribute = true;
+
+	}
+
+	static create( count, itemSize, typeClass = Float32Array ) {
```

Comment:
Maybe instead of using this method we can overload the constuctor? Like so: /ping @aardgoose @sunag 

```js
constructor( array, itemSize, typeClass = Float32Array ) {

    if ( ArrayBuffer.isView( arrray ) === false ) array = new typeClass( array * itemSize );

    super( array, itemSize );

    this.isStorageBufferAttribute = true;

}
```